react-commenting.js

```diff
@@ -112,13 +122,15 @@ class Square extends Component{
 
   handleSquareClick = () => {
     // 12) Here:
+    //This check for the winner on each click to update the board.
     this.props.gamePlay(this.props.index)
```

Comment:
This is the gamePlay method that was passed as props to <Square /> we are giving the method an argument that is the index of whatever square was clicked.